compat/src/index.js

```diff
@@ -117,8 +117,58 @@ const flushSync = (callback, arg) => callback(arg);
  */
 const StrictMode = Fragment;
 
+const startTransition = callback => callback();
+const useTransition = () => {
+	const [isPending, setIsPending] = useState(false);
+
+	const start = cb => {
+		setIsPending(true);
+		cb();
+		setIsPending(false);
+	};
+	return [isPending, start];
+};
+
+const useDeferredValue = val => {
+	const value = useRef(val);
+	return value.current;
```

Comment:
This should return `val` instead. The implementation in this PR would forever return the initial value which is wrong.